examples/tictactoe/src/store/index.ts

```diff
@@ -1,57 +1,70 @@
 import { produce } from "immer";
-import { createZKAppWorkerClient, createZKAssert, createZKState } from "zk-states";
-import type { GameBoard, Player } from "../types";
+import {
+  createZKAppWorkerClient,
+  createZKAssert,
+  createZKState,
+} from "zk-states";
+import type { PlayField, Player } from "../types";
 
-
-const worker = new Worker(new URL("../worker/zkStatesWebWorker.ts", import.meta.url), {
-  type: "module",
-});
+const worker = new Worker(
+  new URL("../worker/zkStatesWebWorker.ts", import.meta.url),
+  {
+    type: "module",
+  },
+);
 
 const workerClient = createZKAppWorkerClient(worker);
 
 //TODO:: use this variable when there are string assertions avaliable
 const zkAssert = createZKAssert(workerClient);
 
-const EMPTY_BOARD = [{ squares: new Array<Player>(9).fill("") }];
-
-// ZK store for the TicTacToe game these values generate a proof on state change
-interface ZKState {
-  board: Player[];
-  history: GameBoard[];
+interface TicTacToeState {
+  board: PlayField[];
   turnNumber: number;
-  xIsNext: boolean;
+  currentPlayer: Player;
   finished: boolean;
+}
 
-  updateBoard: (index: number, value: Player) => void;
-  setBoard: (board: Player[]) => void;
+// ZK store for the TicTacToe game these values generate a proof on state change
+interface ZKState extends TicTacToeState {
+  updateBoard: (index: number) => void;
```

Comment:
```suggestion
  markCell: (index: number) => void;
```
`updateBoard` is quite generic